Makefile

```diff
@@ -74,6 +74,7 @@ test-in-docker:
 	docker run -v "$(shell pwd):/go/src/github.com/aws/amazon-ecs-agent" --privileged "amazon/amazon-ecs-agent-test:make"
 
 run-functional-tests: test-registry
+	docker build -f misc/testnnp/Dockerfile -t "amazon/testnnp:make" misc/testnnp/
```

Comment:
Instead of doing this, add a new target like `netkitten`.  You'll want to make a new `Makefile` in the `testnnp` directory.  You'll also want to update `test-registry` to depend on the new target and update `scripts/setup-test-registry` to ensure that the new image is in the test registry.